lib/bundler/postit_trampoline.rb

```diff
@@ -49,6 +49,14 @@ module BundlerVendoredPostIt; end
   nil
 end
 
+if Gem::Requirement.new(">= 1.13.pre".dup).satisfied_by?(Gem::Version.new(running_version))
+  ENV["BUNDLE_POSTIT_TRAMPOLINING_VERSION"] = installed_version.to_s
+elsif ARGV.empty? || ARGV.any? {|a| %w(install i).include? a }
```

Comment:
@segiddins It was passing fully previously (the only fail for the previous check was codeclimate because of the TODO). This is what made it fail. I was trying to add support for `bundle` command only. It broke a completely different spec, so I'm looking for another way to determine whether the user just runs `bundle` without relying on the `ARGV`.